tests/components/backup/snapshots/test_backup.ambr

```diff
@@ -71,6 +71,10 @@
           'backup_id': 'abc123',
           'database_included': True,
           'date': '1970-01-01T00:00:00.000Z',
+          'extra_metadata': dict({
+            'instance_id': 'our_uuid',
+            'with_automatic_settings': True,
```

Comment:
We could filter out these keys, but it's maybe simpler to just send everything (as the PR does now)

Reply:
Let's send everything.